### PR TITLE
Render -- as is

### PR DESCRIPTION
**Pull request by mehmetoguzderin**

### CODE REVIEW

**Comment by kvark** [comment]
Let's include "++" as we are at it?

**Comment by mehmetoguzderin** [comment]
@kvark I think that is rendered as intended, but if not, what shape is seen over there? Sorry...

**Comment by dneto0** [comment]
I tried it three ways.

First, the original:
```
--
``` 
 original

Then the proposed way
```
&#45;&#45;
```


And a third way using backticks (like other code samples)
```
`--`
```
backtick-minus-minus


I definitely agree the original way is bad because there is no separation visible between the tokens.
I think the backtick method is more readable in the spec source text and does show what is needed.  If the backtick form is acceptable to you, please change to that.

File: wgsl/index.bs [hunk 1]
```diff
@@ -5583,7 +5583,7 @@ The expression must evaluate to a reference with an [=integer scalar=] [=store t
   <tr algorithm="decrement statement">
     <td class="nowrap">|r| : ref&lt;<var ignore>SC</var>,|T|,[=access/read_write=]&gt;,<br>
         |T| is [=integer scalar=]<br>
-    <td class="nowrap">|r|--
+    <td class="nowrap">|r|&#45;&#45;
```
Comment (dneto0) [inline wgsl/index.bs:5586]: I suggest using backticks instead.

See https://github.com/gpuweb/gpuweb/pull/2576#issuecomment-1034377312

**Review by dneto0 — CHANGES_REQUESTED** [review]

File: wgsl/index.bs (hunk 1 above)
Comment (mehmetoguzderin) [reply wgsl/index.bs]: @dneto0 thanks for the feedback, I agree that this looks the best out of three, and I applied the change.

**Review by dneto0 — APPROVED** [review]

**Comment by dneto0** [comment]
Thanks!

**Comment by mehmetoguzderin** [comment]
@kvark I completely agree and that will require a review of spec top to bottom because I think not only there might be other instances but also there will be newly introduced instances as spec approaches final months before its final passes. This PR was just intended to fix the obviously misleading long dash (which is a fair conversion from Bikeshed to do, long dash in normal text is more desired than two consecutive dashes).

**Pull request merged**